### PR TITLE
Frontend/fetch movie details

### PR DESCRIPTION
**Pull request by iljaSL**
fetched movie details to the VideoPlayer component

### CODE REVIEW

File: frontend/src/components/videoPlayer/VideoPlayer.js [hunk 1]
```diff
@@ -24,21 +25,27 @@ const useStyles = makeStyles(() => ({
...
 
-const VideoPlayer = () => {
+const VideoPlayer = (data) => {
   // eslint-disable-next-line max-len
+  const [movie, setMovie] = useState({});
   const classes = useStyles();
   const src = Vid;
 
+  useEffect(async () => {
+    const movieData = await movieService.getMovieData(data.movie.imdbCode);
```
Comment (lapaset) [inline frontend/src/components/videoPlayer/VideoPlayer.js]: I think you need a try-catch block here to catch server errors. Tatiana has a good example in MyProfile :)

File: frontend/src/components/videoPlayer/TitleBanner.js [hunk 2]
```diff
@@ -26,26 +24,26 @@ const useStyles = makeStyles((theme) => ({
     marginTop: '1rem',
   },
 }));
-
-const TitleBanner = () => {
+// eslint-disable-next-line no-undef
```
Comment (lapaset) [inline frontend/src/components/videoPlayer/TitleBanner.js]: I don't think you need this eslint line

File: frontend/src/components/videoPlayer/VideoPlayer.js [hunk 3]
```diff
@@ -24,21 +25,27 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const VideoPlayer = () => {
+const VideoPlayer = (data) => {
   // eslint-disable-next-line max-len
```
Comment (lapaset) [inline frontend/src/components/videoPlayer/VideoPlayer.js]: This is probably some old linter line?

**Review by lapaset — CHANGES_REQUESTED** [review]
I ran the code and everything works perfectly!
I would do some error catching in case server is down/user is not authenticated. After that this should be ready for merge :)

File: frontend/src/components/videoPlayer/VideoPlayer.js (hunk 1 above)
Comment (iljaSL) [reply frontend/src/components/videoPlayer/VideoPlayer.js]: Alright I will check that out in the evening and changed it! :)

**Pull request merged**